Documentation/guides/AndroidAssetPacks.md

```diff
@@ -0,0 +1,176 @@
+# Android Asset Packs
+
+Google Android began supporting splitting up the app package into multiple
+packs with the introduction of the `aab` package format. This format allows
+the developer to split the app up into multiple `packs`. Each `pack` can be
+downloaded to the device either at install time or on demand. This allows
+application developers to save space and install time by only installing
+the required parts of the app initially. Then installing other `packs`
+as required.
+
+There are two types of `pack`. The first is a `Feature` pack, this type
+of pack contains code and other resources. Code in these types of `pack`
+can be launched via the `StartActivity` API call. At this time due to
+various constraints .NET Android cannot support `Feature` packs.
+
+The second type of `pack` is the `Asset` pack. This type of pack ONLY
+contains `AndroidAsset` items. It CANNOT contain any code or other
+resources. This type of `feature` pack can be installed at install-time,
+fast-follow or ondemand. It is most useful for apps which contain allot
+of `Assets`, such as Games or Multi Media applications.
+See the [documentation](https://developer.android.com/guide/playcore/asset-delivery) for details on how this all works.
+.NET Android does not have any official support for this type of pack.
+However a hack is available via the excellent @infinitespace-studios on
+[github](https://github.com/infinitespace-studios/MauiAndroidAssetPackExample).
+This hack allows developers to place additional assets in a special
+`NoTargets` project. This project is built just after the final `aab` is
+produced. It builds a zip file which is then added to the `@(Modules)`
+ItemGroup in the main application. This zip is then included into the
+final app as an additional feature.
+
+## Asset Pack Specification
+
+We want to provide our users the ability to use `Asset` packs without
+having to implement the hack provided by @infinitespace-studios. Using
+a separate project like in the hack is one way to go. It does have some
+issues though.
+
+1. It is a `special` type of project. It requires a `global.json` which imports the
```

Comment:
Extraneous information *here*.  For people who "just found" this page, they're going to want to know:

  * What are asset packs?
  * When do I want to use them?
  * How do I use them?

Implementation details, tradeoffs, alternate solutions, etc., while still useful, should be relegated to the end of the file.